MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+</tr>
+</table>
+
+### DataFormTextStyle
+
+The properties of `Xamarin SfDataForm` [LabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html) are grouped to `DataFormTextStyle` in `.NET MAUI SfDataForm`
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+...
+    xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.LabelTextStyle>
+            <dataForm:DataFormTextStyle
+                TextColor="Red"
+                FontAttributes="Italic"
+                FontSize="12">                
+            </dataForm:DataFormTextStyle>
+        </dataForm:SfDataForm.LabelTextStyle>
+    </dataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight C# %}
+
+using Syncfusion.Maui.DataForm;
+…
+
+SfDataForm dataForm = new SfDataForm();
+this.Content = dataForm;
+this.dataForm.DataObject = new DataModel();
+this.dataForm.LabelTextStyle = new DataFormTextStyle()
+{
+    TextColor = Colors.Red,
+    FontAttributes = FontAttributes.Italic,
+    FontSize = 12,
+};
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+public class DataModel
+{
+	public string Name { get; set; }
+
+    public string Email { get; set; }
+
+    public int PhoneNumber { get; set; }
+
+    public string Address { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>Nil</td>
+<td>TextColor</td>
+<td>Gets or sets the editor text color in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[FontSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html#Syncfusion_XForms_DataForm_LabelStyle_FontSize)'| markdownify }}</td>
+<td>FontSize</td>
+<td>Gets or sets the double value that represents text font size in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[FontFamily](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html#Syncfusion_XForms_DataForm_LabelStyle_FontFamily)'| markdownify }}</td>
+<td>FontFamily</td>
+<td>Gets or sets the string, that represents text font family in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[FontAttributes](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html#Syncfusion_XForms_DataForm_LabelStyle_FontAttributes)'| markdownify }}</td>
+<td>FontAttributes</td>
+<td>Gets or sets the text font attributes in the data form.</td>
+</tr>
+
+</table>
+
+### DataFormViewItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[RowSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItemBase.html#Syncfusion_XForms_DataForm_DataFormItemBase_RowSpan)'| markdownify }}</td>
+<td>RowSpan</td>
+<td>Gets or sets the number of rows that this item spans which is used to increase item's height.</td>
```

Comment:
<td>Gets or sets the number of rows this item spans, which increases the item's height.</td>